packages/sit-onyx/src/components/OnyxDataGrid/features/HeaderCell.vue

```diff
@@ -24,10 +24,11 @@ const slots = defineSlots<{
 .onyx-data-grid-header-cell {
   display: flex;
   align-items: center;
-  gap: var(--onyx-spacing-2xs);
+  gap: var(--onyx-density-2xs);
```

Comment:
```suggestion
  gap: var(--onyx-density-xs);
```

To be aligned with Figma